Makefile

```diff
@@ -15,7 +15,7 @@ ci: deps checkgofmt checkgenerate vet staticcheck ineffassign predeclared test
 
 .PHONY: deps
 deps:
-	go get -d -v -t ./...
+	go get -v -t ./...
```

Comment:
The `-d` flag was deprecated, and the `go get` command was outputting this warning:
```
go: -d flag is deprecated. -d=true is a no-op
```